lib/elixir/lib/kernel/parallel_compiler.ex

```diff
@@ -363,13 +338,12 @@ defmodule Kernel.ParallelCompiler do
         Process.monitor(process)
         wait_for_messages(queue, spawned + 1, waiting, files, result, warnings, state)
 
-      {:struct_available, module} ->
+      {:available, kind, module} ->
```

Comment:
Minor: why `:struct_available` is changed to `{:available, :struct}` but `:module_available` not to `{:available, :module}`? 🙂

Reply:
The module_available passes a bunch of extra information which makes it unique. The :struct mechanism though is simpler and it has been generalized for other use cases. I will effectively use it in an upcoming PR. :)